crates/sargon/src/wrapped_radix_engine_toolkit/low_level/v2/subintent.rs

```diff
@@ -3,20 +3,24 @@ use delegate::delegate;
 use std::hash::Hasher;
 
 #[derive(Clone, PartialEq, Eq, derive_more::Debug)]
-#[debug("{:?}", self.intent_core)]
+#[debug("header:\n{:?}\n\nmessage:\n{:?}\n\nmanifest:\n{}\n\n", self.header, self.message, self.manifest.manifest_string())]
 pub struct Subintent {
-    pub intent_core: IntentCoreV2,
+    pub header: IntentHeaderV2,
```

Comment:
Subintent was incorrectly represented through TransactionManifestV2, it should be SubintentManifest.

Reply:
nope, removed.